core/deployment/src/main/java/io/quarkus/deployment/logging/LoggingResourceProcessor.java

```diff
@@ -12,11 +13,7 @@
 import io.quarkus.deployment.annotations.BuildStep;
 import io.quarkus.deployment.annotations.ExecutionTime;
 import io.quarkus.deployment.annotations.Record;
-import io.quarkus.deployment.builditem.LogCategoryBuildItem;
-import io.quarkus.deployment.builditem.LogConsoleFormatBuildItem;
-import io.quarkus.deployment.builditem.LogHandlerBuildItem;
-import io.quarkus.deployment.builditem.RunTimeConfigurationDefaultBuildItem;
-import io.quarkus.deployment.builditem.SystemPropertyBuildItem;
+import io.quarkus.deployment.builditem.*;
```

Comment:
We don't use star imports. Make sure to configure your IDE to use the https://github.com/quarkusio/quarkus/blob/master/ide-config/eclipse-format.xml format file

Reply:
Done